package.json

```diff
@@ -177,9 +177,10 @@
     "redux": "^3.6.0",
     "redux-thunk": "^2.2.0",
     "reselect": "^2.5.4",
+    "scrypt": "^6.0.3",
```

Comment:
I don't know why exactly, but adding this was necessary after updating to web3 1.0.

Reply:
This is a pretty rough dependency (assuming it wasn't already there through some other dependency). `scrypt` requires compilation, and even though we're putting out binaries of Ganache, more compilation with JS is never a good thing. If there's a chance to remove it, I would (again, if it wasn't an indirect dependency to begin with).

Reply:
It's a transitive dependency, anyway - `web3-eth-accounts` requires `scrypt` and `scryptsy`

Reply:
I'm sure you trust me, but here's the dependency tree after I removed this line: https://gist.github.com/benjamincburns/62064d5f6dae3202b846ae143c254f49